src/main/java/com/rocketden/main/exception/api/ApiErrorResponse.java

```diff
@@ -3,13 +3,15 @@
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 
-// Response object returned to client containing message
+// Response object returned to client containing message and error type (name of enum var)
 @Getter
 @EqualsAndHashCode
 public class ApiErrorResponse {
     private final String message;
+    private final String type;
```

Comment:
Nit: should error type go before the message?

Reply:
Sorry, just realized I totally forgot about this. It looks like by default, the JSON responses from these types of classes are unordered; however, there is a way to make them ordered using the annotation explained [here](https://stackoverflow.com/questions/9610948/json-fields-unordered). I added a backlog card in case we want to do this for all of our response objects.